rest/taskana-rest-spring/pom.xml

```diff
@@ -14,6 +14,18 @@
 		<relativePath>../pom.xml</relativePath>
 	</parent>
 
+	<dependencyManagement>

```

Comment:
As above: Is there any reason for this?

Reply:
lets postpone this to another ticket. tried to get rid of it.